src/cmsAdapter.js

```diff
@@ -8,6 +8,20 @@ const cmsAdapter = {
     const payload = await res.json();
     return payload.data;
   },
+  async loadUpcomingScreenings(id) {
+    const res = await fetch(
+      API_BASE + '/screenings?populate=movie&filters[movie]=' + id
+    );
+    const payload = await res.json();
+    return payload;
+  },
+  async loadAllReviews() {
+    const response = await fetch(
+      'https://plankton-app-xhkom.ondigitalocean.app/api/reviews?populate=movie'
```

Comment:
Why not use `API_BASE` here? Staying consistent about things like this is a great way to avoid future bugs. Otherwise, changing the value `API_BASE` would work in most places, and it can be very difficult to realize that it has no effect in a few places that hardcode the URL (like here).